index.html

```diff
@@ -0,0 +1,228 @@
+          </div>
+          <div class="col-4">
+            <img src="img/check-circle.png" alt="" />
+            <h2 class="travel-bigger">Select Places</h2>
+          </div>
+          <div class="col-4">
+            <img src="img/thumbs-up.png" alt="" />
+            <h2 class="travel-bigger">Book & Enjoy</h2>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="travel__book1">
+      <div class="container">
+        <div class="row">
+          <div class="col-6">
+            <div class="travel-spain__book1">
+              <h5 class="travel-text__verysmall">Spain</h5>
+              <h3 class="travel-text__middle">Grand Spain Madrid</h3>
+              <h4 class="travel-text__small">
+                But I must explain to you how all this mistaken idea of <br />
+                account to denouncing pleasure and praising pain was <br />
+                account born and I account will give you a complete <br />
+                account of the system
+              </h4>
+              <button class="travel-button__books">Book Now</button>
+            </div>
+          </div>
+          <div class="col-6"></div>
+        </div>
+      </div>
+    </div>
+    <div class="travel__book2">
+      <div class="container">
+        <div class="row">
+          <div class="col-6"></div>
+          <div class="col-6">
+            <div class="travel-spain__book2">
+              <h5 class="travel-text__verysmall">Spain</h5>
+              <h3 class="travel-text__middle">Grand Spain Madrid</h3>
+              <h4 class="travel-text__small">
+                But I must explain to you how all this mistaken idea of <br />
+                account to denouncing pleasure and praising pain was <br />
+                account born and I account will give you a complete <br />
+                account of the system
+              </h4>
+              <button class="travel-button__books">Book Now</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="travel__whattouristsay">
+      <div class="container">
+        <h2 class="travel-text__bigger-last">What Our Tourists Say</h2>
+        <div class="sam">
+          <img class="back" src="img/ReviewPhoto.png" alt="" />
+          <div class="whattouristsay-textfield">
+            <h3 class="travel-text__middle">Sam Sumon</h3>
+            <h4 class="travel-text__small">
+              But I must explain to you how all this mistaken idea of account to
+              denouncinpleasure <br />
+              and praising pain But I must explain to you how all this mistaken
+              idea of account to nai <br />
+              denouncing pleasure and praising pain was account born and I
+              account will give you a <br />
+              complete account of the system was account born and I account will
+              give you a nai the <br />
+              complete account of the system
+            </h4>
```

Comment:
`<h1>` - `<h6>` - только для заголовков, а у тебя здесь цитата
```suggestion
            <blockquote class="travel-text__small">
              But I must explain to you how all this mistaken idea of account to
              denouncinpleasure <br />
              and praising pain But I must explain to you how all this mistaken
              idea of account to nai <br />
              denouncing pleasure and praising pain was account born and I
              account will give you a <br />
              complete account of the system was account born and I account will
              give you a nai the <br />
              complete account of the system
            </blockquote>
```